doc/README.md

```diff
@@ -189,8 +192,46 @@ paths with the highest cum weight.
 * **-svg:** Generates a report in SVG format.
 * **-web:** Generates a report in SVG format on a temp file, and starts a web
   browser to view it.
-* **-png, -jpg, -gif, -pdf:** Generates a report in these formats,
+* **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
+
+### Interpreting the Callgraph
+
+* **_flat_ value**: the value of a location itself, is indicated by the box size.
+
+    *   Nodes with larger box have larger _flat_ values.
```

Comment:
This documentation page seems to occasionally use "value" and "weight" terms interchangeably. It would be good to gradually become more consistent on this.

Reply:
Changed to use "value" consistently throughout the doc.